src/driver/sqljs/SqljsQueryRunner.ts

```diff
@@ -8,12 +8,12 @@ import {QueryFailedError} from "../../error/QueryFailedError";
  * Runs queries on a single sqlite database connection.
  */
 export class SqljsQueryRunner extends AbstractSqliteQueryRunner {
-    
+
```

Comment:
Probably best to avoid these changes in unrelated files at least

Reply:
OK, may be i need close this proposal and open new? It`s my first proposal and i wanted to fix some style errors too.

Reply:
In any project it is generally better to keep changes separate to make it easier to find bugs, explanations for changes, etc. You can fix style errors that are right beside the code that you are changing but you should avoid doing it elsewhere. Best way to do this is to create a new branch on your repo specifically for this change, and only push the relevant commits there. Then when the pull request is being reviewed, the reviewers can focus only on the main problem and it can get merged quicker.